docs/minio-admin-complete-guide.md

```diff
@@ -413,9 +413,12 @@ NAME:
   mc admin heal - heal disks, buckets and objects on MinIO server
```

Comment:
sh tag is causing highlighting of  keywords select, do, in, and help , several instances throughout document

Reply:
Changed `     ```sh -> ``` `  everywhere.